pkgs/development/interpreters/python/tests.nix

```diff
@@ -133,6 +134,19 @@ let
     # test-overrideScope = let
     #  myPackages = python.pkgs.overrideScope extension;
     # in assert myPackages.foobar == myPackages.numpy; myPackages.python.withPackages(ps: with ps; [ foobar ]);
+  } // lib.optionalAttrs (python ? pythonAttr) {
+    # Test applying overrides using pythonPackagesOverlays.
+    # Test is commented out because we should not use pkgs.extend in Nixpkgs
```

Comment:
This test doesn't look commented out. I think it would be fine to test it like this, `.extend` isn't a huge cost I'm pretty sure

Reply:
Ah yes, I uncommented this to test whether it was still functioning.

In the past usages of `pkgs.extend` were removed.